ui/v2.5/src/components/Tagger/Tagger.tsx

```diff
@@ -355,15 +355,12 @@ const TaggerList: React.FC<ITaggerListProps> = ({
       return (
         <div key={scene.id} className="my-2 search-item">
           <div className="row">
-            <div className="col-md-6 my-1 text-truncate align-self-center">
-              <Link
-                to={`/scenes/${scene.id}`}
-                className="scene-link"
-                title={scene.path}
-              >
-                {originalDir}
-                <wbr />
-                {`${file}.${ext}`}
+            <div className="col-md-6 my-1 align-self-center">
+              <Link to={`/scenes/${scene.id}`} className="scene-link">
+                <TruncatedText
+                  text={`${originalDir}{file}.${ext}`}
```

Comment:
I came here to comment on one thing, but I found a second one.
1. The `ext` variable already contains the dot (`.mp4`), so the dot is always duplicated here.
2. One `$` was removed here.
```suggestion
                  text={`${originalDir}${file}${ext}`}
```

Reply:
Thanks, I've noticed the double periods before but somehow I never thought to look for it.